homeassistant/components/zwave_js/const.py

```diff
@@ -33,3 +33,5 @@
 ATTR_PROPERTY_NAME = "property_name"
 ATTR_PROPERTY_KEY_NAME = "property_key_name"
 ATTR_PARAMETERS = "parameters"
+
+THERMOSTAT_MODE_PROPERTY = "mode"
```

Comment:
This one might belong into `zwave_js_server`?

Reply:
yes

Reply:
Ok. Opened a PR: https://github.com/home-assistant-libs/zwave-js-server-python/pull/94

Reply:
Merged. We'll include it in a release very soon.

Reply:
Thanks Martin, I'll add those changes as soon as the related PR in HA and rebase.

Reply:
It's in HA now, you can rebase.